### PR TITLE
User logins

### PR DESCRIPTION
**Pull request by bnmajor**
Updates the Ansible playbook and associated parameters and docs:
- Match the deployed folder/collection naming and structure as well as the default admin user. Removes the creation of a non-admin user.
- Require email verification and admin approval for new users
- Pass in required SMTP information to setup email service
- Update docs to reflect these changes

### CODE REVIEW

File: devops/ansible/site.yml [hunk 1]
```diff
@@ -4,106 +4,179 @@
...
-          firstName: "esimmonadmin"
-          lastName: "esimmonadmin"
-          login: "esimmonadmin"
-          password: "{{ admin_password }}"
-          email: "esimmonadmin@kitware.com"
+          firstName: 'esimmon'
+          lastName: 'esimmon'
+          login: 'esimmon'
+          password: '{{ admin_password }}'
+          email: 'brianna.n.major@gmail.com'
```
Comment (cjh1) [inline devops/ansible/site.yml:26]: Not sure you want to include this? You might get alot of spam ( may be too late! )

File: devops/ansible/site.yml [hunk 2]
```diff
@@ -4,106 +4,179 @@
...
+          'key': 'core.smtp.encryption',
+          'value': starttls
+        }
+        - {
+          'key': 'core.email_from_address',
+          'value': 'eSimMon Dashboard <no-reply@esimmon.kitware.com>'
+        }
+        - {
+          'key': 'core.email_host',
+          'value': 'api.esimmon.kitware.com'
```
Comment (cjh1) [inline devops/ansible/site.yml:129]: This should probably go in our deployment repo ...

**Review by cjh1 — COMMENTED** [review]
Why ' => " seems like most playbooks use ", any particular reason?

**Comment by bnmajor** [comment]
> Why ' => " seems like most playbooks use ", any particular reason?

No good reason other than there was a mix and I wanted some consistency. Updated to use `"` now instead of `'` since it makes sense to follow the common pattern found in other playbooks.

File: devops/ansible/site.yml [hunk 3]
```diff
@@ -4,35 +4,35 @@
...
+        host: "{{ girder_host }}"
+        port: "{{ girder_port }}"
+        scheme: "{{ girder_scheme }}"
         user:
-          firstName: "esimmonadmin"
-          lastName: "esimmonadmin"
-          login: "esimmonadmin"
+          firstName: "esimmon"
+          lastName: "esimmon"
+          login: "esimmon"
```
Comment (cjh1) [inline devops/ansible/site.yml:24]: Why the rename, just to be more concise ?

File: devops/ansible/site.yml [hunk 4]
```diff
@@ -41,51 +41,124 @@
...
+        host: "{{ girder_host }}"
+        port: "{{ girder_port }}"
+        scheme: "{{ girder_scheme }}"
+        username: "esimmon"
         password: "{{ admin_password }}"
         collection:
           name: "eSimMon"
-          description: "eSimMon collection"
-          public: "{{ public_data | default('yes') }}"
+          description: ""
```
Comment (cjh1) [inline devops/ansible/site.yml:141]: Any reason to remove the description?

**Review by cjh1 — APPROVED** [review]
LGTM

**Pull request merged**